src/sentry/api/event_search.py

```diff
@@ -722,6 +722,7 @@ def to_list(value):
     "min": {"snuba_name": "min", "fields": ["time", "timestamp", "transaction.duration"]},
     "max": {"snuba_name": "max", "fields": ["time", "timestamp", "transaction.duration"]},
     "avg": {"snuba_name": "avg", "fields": ["transaction.duration"]},
+    "sum": {"snuba_name": "sum", "fields": ["time", "timestamp", "transaction.duration"]},
```

Comment:
I'm not sure suming up the time or timestamp columns is as useful as the duration. We might want to start with only duration.

Reply:
I agree. timestamp are just unix timestamps; and summing them up isn't actually useful.

Just having `["transaction.duration"]` for the fields would be sufficient.